CHANGES.rst

```diff
@@ -0,0 +1,13 @@
+Changes
+=======
+
+Version 0.2.0 (2018-04-17)
```

Comment:
Please leave version and date on `FIXME` as we shall be releasing for later versions...